README.md

```diff
@@ -1,3 +1,226 @@
-# HTTP Middleware
+HTTP Middleware
```

Comment:
Was going to suggest changing the above to "HTTP Server Middleware", and then realized #25 is a PR against this branch, doing that...